src/searching.md

```diff
@@ -183,148 +182,151 @@ Searches that ignore combining characters are slower than regular searches.
 
-Start a search with `re:` to search by regular expression. To make things easier, Anki will
+Start a search with `re:` to search using regular expression. To make things easier, Anki will
 treat the following as [raw input](#raw-input), so bear in mind the rules listed there.
 
 Some examples:
 
 `"re:(some|another).*thing"`\
-find notes that have "some" or "another" on them, followed by 0 or more characters, and then "thing"
+finds notes that have "some" or "another" on them, followed by 0 or more characters, and then "thing".
 
 `re:\d{3}`\
-find notes that have 3 digits in a row
+finds notes that have 3 digits in a row.
 
 Regular expressions can also be limited to a specific field. Please note that unlike the normal searches
-in a specific field, regular expressions in fields don't require an exact match. Eg:
+in a specific field, regular expressions in fields don't require an exact match:
 
 `front:re:[a-c]1`\
-matches uppercase or lowercase a1, B1 or c1 that occurs anywhere in the "Front" field
+matches uppercase or lowercase a1, B1 or c1 that occurs anywhere in the "Front" field.
 
 `front:re:^[a-c]1$`\
-like the above, but will not match if any other text falls before or after a1/b1/c1.
+same as previous, but will not match if any other text falls before or after a1/b1/c1.
 
-Anki 2.1.50 added regex support for tags:
+Anki 2.1.50+ supports regular expressions for tags:
 
 `tag:re:^parent$`\
-find notes with the exact tag "parent", disregarding any child tags like "parent::child"
+finds notes with the exact tag "parent", disregarding any child tags like "parent::child".
 
 `"tag:re:lesson-(1[7-9]|2[0-5])"`\
-find notes with tags "lesson-17" through "lesson-25"
+finds notes with tags "lesson-17" through "lesson-25".
 
-You can learn more about regular expressions here: <https://regexone.com/lesson/introduction_abcs>
+For more information on regular expressions, see [this website](<https://regexone.com/lesson/introduction_abcs>).
 
 Some things to be aware of:
 
 - The search is case-insensitive by default; use `(?-i)` at the start to turn on case sensitivity.
 - Some text like spaces and newlines may be represented differently in HTML - you can
   use the HTML editor in the editing screen to see the underlying HTML contents.
-- For the specifics of Anki's regex support, please see the regex crate documentation: <https://docs.rs/regex/1.3.9/regex/#syntax>
+- For the specifics of Anki's regex support, see the [regex crate documentation]
+(<https://docs.rs/regex/1.3.9/regex/#syntax>)
 
 ## Card state
 
 `is:due`\
-review cards and learning cards waiting to be studied
+review cards and learning cards waiting to be studied.
 
 `is:new`\
-new cards
+new cards.
 
 `is:learn`\
-cards in learning
+cards in learning.
 
 `is:review`\
-reviews (both due and not due) and lapsed cards
+reviews (both due and not due) and lapsed cards.
 
 `is:suspended`\
-cards that have been manually suspended
+cards that have been automatically or manually suspended.
```

Comment:
Can this line and the next (241) match? Either phrasing is fine.

Reply:
I was about to follow your advice here, but I remembered that some of the variability is intentional. It is a bit boring to read a text if everything seems to be using some sort of parallel syntax. I don't think this should be changed.

Reply:
Responded above at line 41.

Reply:
I have resolved the other conversation as the explanation itself was confusing. For this one, I want to ask @dae why the original phrasing was different for is:suspended and is:burued? The docs are created in your vision so I'd like to get your feedback first.

Reply:
I can't remember any specific reason for the differing wording, and don't have strong feelings here.